index.html

```diff
@@ -60,5 +60,387 @@ <h2>I am a Software Developer</h2>
+                sign-ups required. has been the<br />
+                industry's standard
+              </p>
+              <ul class="d-flex">
+                <li>
+                  <a href="#">
+                    <img
+                      alt="htmlButton"
+                      src="img/htmlTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="bootstrapButton"
+                      src="img/bootstrapTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="rubyButton"
+                      src="img/rubyTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+              </ul>
+              <a href="#">
+                <img
+                  alt="seeProjectButton"
+                  src="img/Action.png"
+                  class="buttonProperties"
+                />
+              </a>
+            </div>
+            <div class="gridItem grid8 backGroundImg">
+              <h2>
+                Profesional Art<br />
+                Printing Data
+              </h2>
+              <p>
+                A daily selection of privately<br />
+                personalized reads; no accounts or<br />
+                sign-ups required. has been the<br />
+                industry's standard
+              </p>
+              <ul class="d-flex">
+                <li>
+                  <a href="#">
+                    <img
+                      alt="htmlButton"
+                      src="img/htmlTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="bootstrapButton"
+                      src="img/bootstrapTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="rubyButton"
+                      src="img/rubyTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+              </ul>
+              <a href="#">
+                <img
+                  alt="seeProjectButton"
+                  src="img/Action.png"
+                  class="buttonProperties"
+                />
+              </a>
+            </div>
+          </div>
+        </section>
+        <section class="aboutMe">
```

Comment:
Please try to follow the design as much as possible 🧐 As you can see in the following image, we can improve the margins in your design so that it resembles more to Figma's

Your design 

![image](https://user-images.githubusercontent.com/15807343/153959214-eb7f9960-b32e-40c8-a5e8-a75ca2ceb5a4.png)

Figma's 
![Captura](https://user-images.githubusercontent.com/15807343/153959297-9f22dbe6-4a23-4890-a4aa-cd43527af9f2.PNG)

Also, kindly use the same fonts on your design! That will take your portfolio to the next level 🔥